FinnUI/Sources/Fullscreen/NotificationCenter/Views/NotificationCenterView.swift

```diff
@@ -254,6 +259,18 @@ extension NotificationCenterView: UIScrollViewDelegate {
        reloadOnEndDragging = false
        delegate?.notificationCenterView(self, segment: selectedSegment, didPullToRefreshUsing: refreshControl)
     }
+
+    public func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        let offsetY = scrollView.contentOffset.y
+        let contentHeight = scrollView.contentSize.height
+
+        let hasScrolledEnough = offsetY > contentHeight - scrollView.frame.height * 4
```

Comment:
I am not sure about this constant name. I'm open to suggestions

Reply:
Hmm, not sure. The only other thing I can think of is `willReachEndOfContent`, but not sure if that is better

Reply:
sounds better to me at least, regarding intent